docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -1055,8 +1055,79 @@ Because `MediaLibrary` is the `TvLibrary` class parent, a user with the `admin`
 
 CAUTION: Annotation-based permissions do not work with custom xref:security-customization.adoc#jaxrs-security-context[Jakarta REST SecurityContexts] because there are no permissions in `jakarta.ws.rs.core.SecurityContext`.
 
+[[permission-checker]]
+==== Create permission checkers
+
+By default, `SecurityIdentity` must be augmented with permissions which can be used to check if this identity passes `@PermissionAllowed` authorization restrictions.
```

Comment:
I don't think using "augmented" is great here. (Augmented is the augmentation phase of Quarkus, and it's in the log of each build). "configured" might be better.

Reply:
it can _only_ (technically that's not true, but users don't use http perms for that I think) happen in `SecurityIdentityAugmentor`, that is why I used `augmented`. I am perfectly fine with `configured`, I'll use it.

Reply:
Thanks! I think _configure_ is less confusing in this context.

I now remember about the `SecurityIdentityAugmentor` discussions.... At that time, everything looked like an augmentation :-)